### ISSUE TITLE
feature requests

### ISSUE THREAD

**Issue by ChangePlaces** [shaarli/Shaarli#1432]
A few little suggestions!

- the ability to add tags when adding the url, rather than being taken to a second page (with tag autocomplete)
- after adding a url, focus on an input where pressing enter will allow the bookmark to go ahead, instead of a description where tabbing out is awkward
- view tagged bookmarks as contents inside folders named after the tag, so tag cloud, because a list of tags with a folder icon and tag title underneath rather than a squished up bunch of words. Then, clicking on a 'folder' takes you to a filtered search for that tag.

**Comment by nodiscc** [comment]
> the ability to add tags when adding the url, rather than being taken to a second page (with tag autocomplete)

You can already do that by accessing https://my.shaarli.domain/?post=# or you can use one of the [browser extensions](https://shaarli.readthedocs.io/en/master/Community-&-Related-software/#browser-addons) (I almost never use the `+Shaare` button directly from Shaarli except for posting notes). If you want to suggest improvements to that, please open a separate issue.

> a url, focus on an input where pressing enter will allow the bookmark to go ahead, instead of a description

Then it should be optional/selectable, the current default focus in the `Description` field is fine for me, because I always set some tags on new bookmarks, and the tags field is just one `<TAB>` key away. In fact I argue that having `Tags` the default would be better. Can you open a separate issue for this?

> , clicking on a 'folder' takes you to a filtered search for that tag.

Already exists, use one of the list views in the tag cloud (https://demo.shaarli.org/?do=taglist&sort=alpha, https://demo.shaarli.org/?do=taglist&sort=usage). If you want to suggest improvements, please open a separate issue.

In short, please open separate issues for each request :)

**Comment by ChangePlaces** [comment]
if you have the ability to close my issue, you have the ability to create other issues. thanks for making work for me. i've since decided to move on, but good luck with the project.

**Comment by nodiscc** [comment]
> if you have the ability to close my issue, you have the ability to create other issues

I opened #1435 #1436 #1437, thanks for taking the time to report this.
